library/hmac.po

```diff
@@ -3,32 +3,35 @@
 # This file is distributed under the same license as the Python package.
 #
 # Translators:
+# Adrian Liaw <adrianliaw2000@gmail.com>, 2018
+# Phil Lin <linooohon@gmail.com>, 2022
 msgid ""
 msgstr ""
 "Project-Id-Version: Python 3.10\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2020-06-20 18:08+0800\n"
-"PO-Revision-Date: 2018-05-23 16:03+0000\n"
-"Last-Translator: Adrian Liaw <adrianliaw2000@gmail.com>\n"
+"PO-Revision-Date: 2022-03-14 23:58+0800\n"
+"Last-Translator: Phil Lin <linooohon@gmail.com>\n"
 "Language-Team: Chinese - TAIWAN (https://github.com/python/python-docs-zh-"
 "tw)\n"
 "Language: zh_TW\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=1; plural=0;\n"
+"X-Generator: Poedit 3.0.1\n"
 
 #: ../../library/hmac.rst:2
 msgid ":mod:`hmac` --- Keyed-Hashing for Message Authentication"
-msgstr ""
+msgstr ":mod:`hmac` --- 基於鍵散列的訊息驗證"
```

Comment:
Please check the whole translation in this PR for the two following issues:

- As we discussed in Discord, translate the term `hash` as `雜湊` instead of `散列`.
- The term `key` here refers to the authentication key rather than the key in the hash.
 
```suggestion
msgstr ":mod:`hmac` --- 基於金鑰雜湊的訊息驗證"
```

Reply:
Thanks for the review!